messaging/src/main/java/org/axonframework/queryhandling/FluxSinkWrapper.java

```diff
@@ -18,59 +18,53 @@
 
 import reactor.core.Disposable;
 import reactor.core.publisher.FluxSink;
+import reactor.core.publisher.Sinks;
 
 /**
- * Wrapper around {@link FluxSink}. Since project-reactor is not a required dependency in this Axon version, we need
+ * Wrapper around {@link Sinks.Many}. Since project-reactor is not a required dependency in this Axon version, we need
  * wrappers for backwards compatibility. As soon as dependency is no longer optional, this wrapper should be removed.
  *
  * @param <T> The value type
  * @author Milan Savic
+ * @author Stefan Dragisic
  * @since 3.3
  */
 class FluxSinkWrapper<T> {
 
-    private final FluxSink<T> fluxSink;
+    private final Sinks.Many<T> fluxSink;
```

Comment:
Nit: I would even rename the class and parameter here. It's all package-private, so free game to more closely resemble what's being wrapped.

Reply:
I was thinking about that but found FluxSink more descriptive than some other name (like SinksManyWrapper)

Reply:
True, true, it's a "sink of many results", essentially a flux sink. Guessing they removed the notion of flux as it's project-reactor specific?

Reply:
not sure, but I don't think that's the reason